.bazelrc

```diff
@@ -16,6 +16,7 @@ build --ios_simulator_version=14.4
 build --spawn_strategy=local
 build --verbose_failures
 build --workspace_status_command=envoy/bazel/get_workspace_status
+build --xcode_version_config=//ci:xcode_config
```

Comment:
to avoid requiring folks having 12.4 locally can you move this to the `build:remote-ci-macos` config?

Reply:
the project already requires everyone to have 12.4 because of `--xcode_version=12.4` below, but sure will do